client/src/components/Modal.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Modal as ModalBootstrap } from "react-bootstrap";
+
+const Modal = ({ children, isOpen, modalClose, dialogClassName, centered }) => {
+  return (
+    <ModalBootstrap
+      dialogClassName={dialogClassName}
+      centered={centered}
+      show={isOpen}
+      onHide={modalClose}
+    >
+      <ModalBootstrap.Header
+        className="border-bottom-0"
+        closeButton
+      ></ModalBootstrap.Header>
```

Comment:
```suggestion
      />
```

since this component has no children, you can close it within the same <>

Reply:
Absolutely.